CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Please document your changes in this format:
 ```
 
 ## [Unreleased]
+### Changed
+- remove invitation by e-mail at members page and substitute it for a group link to be copied @brnsolikyl [#2349]
```

Comment:
I would just write it as the PR title `Remove invitation by e-mail at members page` - more info is also in the PR